### DISCUSSION TITLE
mmyolo如何在训练多类目标检测模型后，查看单类的AP50？

### DISCUSSION THREAD

**Discussion by Jerry-Li1998** [open-mmlab/mmyolo#560] [Q&A]
默认输出好像是mAP，mmdetection里面好像只是修改mmdet/datasets/coco.py里面的iou_thrs。不知道mmyolo需要如何修改，ask for help!

**Comment by hhaAndroid** [comment]
@Jerry-Li1998 Same usage as mmdet.

`python tools/test.py config ckpt --cfg-options val_evaluator.classwise=True`
